readme.md

```diff
@@ -119,6 +119,20 @@ checkPassword('foo');
 //=> ArgumentError: Expected string `password` to have a minimum length of `6`, got `foo`
 ```
 
+### ow.method(predicates, label, body)
+
+Wrap a function with parameter validation. Useful for writing strongly-typed functions which will be called with untrusted input.
```

Comment:
Would be useful to elaborate a little bit more on the use-case. As a user, I'm not entirely sure still when I would use it.